.github/workflows/minikube.yml

```diff
@@ -0,0 +1,172 @@
+---
+name: MiniKube Heml Deployment
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  job1:
+    runs-on: ubuntu-latest
+    name: build and deploy to minikube
+    steps:
+      - uses: actions/checkout@v2
+
+      # HELM Config
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.10.0
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+
+      - run: mkdir logs
+
+      # Install dependencies
+      - run: npm ci
+
+      # Create certs
+      - run: sudo chown -R runner:docker certs
+      - run: npm run certs
+
+      # Remove route set for openshift
+      - name: remove route config
+        run: sudo rm helm/bailo/templates/route.yaml
```

Comment:
Don't remove the `route.yaml` file, instead add a `value.yaml` entry to disable it.

Reply:
route is not optional